tree_migration_milestone1.js

```diff
@@ -22,6 +22,7 @@ async function migrate() {
       for (let entry of attributes) {
         attributes_in_json_format.entries.push(entry);
       }
+      return attributes_in_json_format;
```

Comment:
This return was missing and was causing failures when trees with corresponding tree_attributes had data. Sorry I missed to notice that in the original code review. This fixes it.